### PR TITLE
Improvements for framecryptor

### PR DESCRIPTION
**Pull request by cloudwebrtc**

### CODE REVIEW

File: lib/src/participant/local.dart [hunk 1]
```diff
@@ -200,34 +200,36 @@ class LocalParticipant extends Participant<LocalTrackPublication> {
...
-            codec: publishOptions.videoCodec,
-            cid: track.getCid(),
-            enableSimulcastLayers: true),
+          codec: publishOptions.videoCodec,
+          cid: track.getCid(),
+        ),
         lk_rtc.SimulcastCodec(
-            codec: publishOptions.backupCodec!.codec.toLowerCase(),
-            cid: '',
-            enableSimulcastLayers: publishOptions.backupCodec!.simulcast),
```
Comment (davidzhao) [inline lib/src/participant/local.dart:209]: these aren't required anymore?

**Review by davidzhao — APPROVED** [review]

File: lib/src/participant/local.dart (hunk 1 above)
Comment (cloudwebrtc) [reply lib/src/participant/local.dart:209]: When I upgraded the protocol, enableSimulcastLayers seemed to be removed

Comment (cloudwebrtc) [reply lib/src/participant/local.dart:209]: https://github.com/livekit/protocol/commit/13abb13b71b7461d04be5b651a9938454655bf2c

Comment (davidzhao) [reply lib/src/participant/local.dart:209]: got it. I guess we don't use it on the server side.

**Pull request merged**